pkg/controller/certificates/sync.go

```diff
@@ -290,6 +301,10 @@ func (c *Controller) updateSecret(crt *v1alpha1.Certificate, namespace string, c
 	// if it is a new resource
 	if secret.SelfLink == "" {
 		secret, err = c.Client.CoreV1().Secrets(namespace).Create(secret)
+		enableOwner := c.CertificateOptions.EnableOwnerRef
+		if enableOwner {
+			secret.SetOwnerReferences(append(secret.GetOwnerReferences(), ownerRef(crt)))
```

Comment:
We need to set this field *before* calling Create on the resource, else the field won't actually be set.

Reply:
Thanks! The filed is now configured before create operation.